python/tests/unit/arcticdb/version_store/test_row_range.py

```diff
@@ -89,3 +91,31 @@ def test_row_range_pickled_symbol(lmdb_version_store):
     assert lmdb_version_store.is_symbol_pickled(symbol)
     with pytest.raises(InternalException):
         _ = lmdb_version_store.read(symbol, row_range=(1, 2))
+
+
+@pytest.mark.parametrize("row_range,expected", (
+    ((-5, None), pd.DataFrame({"a": np.arange(95, 100)})),
+    ((5, None), pd.DataFrame({"a": np.arange(5, 100)})),
+    ((0, None), pd.DataFrame({"a": np.arange(0, 100)})),
+    ((None, -5), pd.DataFrame({"a": np.arange(95)})),
+    ((None, 5), pd.DataFrame({"a": np.arange(5)})),
+    ((None, 0), pd.DataFrame({"a": []}, dtype=np.int64)),
+    ((None, None), pd.DataFrame({"a": np.arange(100)})),
```

Comment:
Looking at `RowRangeClause`
```
util::check(start || end, "Expect at least one of start and end to be present");
```
(None, None) is supposed to throw but the test passes. What am I missing?